website/templates/index.html

```diff
@@ -0,0 +1,218 @@
+{% import "_macros.html" as macros %}
+<!DOCTYPE html>
+<html lang="{{ lang }}">
+  <head>
+    <meta charset="UTF-8" />
+    <title>{% block title %}{{ config.title }}{% endblock title %}</title>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    {% block favicon %}
+    <link rel="icon" href="{{ config.base_url }}favicon.ico" />
+    {% endblock favicon %} {% include "_variables.html" %} {% block fonts %}
+    <link
+      href="https://fonts.googleapis.com/css?family=Alfa+Slab+One&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Fira+Sans:400,500,600&display=swap"
+      rel="stylesheet"
+    />
+    {% endblock fonts %}
+    <link rel="stylesheet" href="{{ get_url(path="prism.css") }}" />
+    <link rel="stylesheet" href="{{ get_url(path="normalize.css") }}" />
+    <link rel="stylesheet" href="{{ get_url(path="juice.css") }}" />
+    <link rel="stylesheet" href="{{ get_url(path="syntax-theme.css") }}" />
+  </head>
+  <body>
+    {% block header %}
+
+    <header class="pos-absolute" style="background-color: transparent">
+      {{ macros::render_header() }}
+    </header>
+
+    <div class="hero">
+      {% block hero %}
+      <script async defer src="https://buttons.github.io/buttons.js"></script>
+      <section class="text-center">
+        <h1
+          class="heading-text"
+          style="font-size: 50px; color: var(--header-text-color)"
+        >
+          Minecraft servers for the <br />
+          modern age
+        </h1>
+        <h3 class="title-text" style="color: var(--header-text-color)">
+          <b>Valence</b> is A Rust framework for building modular <br />
+          and high performance Minecraft servers.
```

Comment:
The text on the left is quite orange. I tried to break it up a bit by making this sentence white but wasn't sure how to do that.

Reply:
Looks a little odd like this, doesn't it?
![grafik](https://github.com/valence-rs/valence/assets/75166328/37078da7-c307-4974-a8ef-279eb53200a2)

Reply:
I suppose so, but I'd still say it's an improvement? Whatever color it ends up being, I do think it should be visually distinct from the text above.

Maybe you could make the "Quick Start" button the red from the logo? Idk if that would look silly.

Reply:
Maybe the badges could be placed on the header to the right of "Valence"? Just throwing out some ideas.

Reply:
> Maybe the badges could be placed on the header to the right of "Valence"? Just throwing out some ideas.

So they should be on every page?

Reply:
Yeah that would be a bit strange.

Reply:
I think it looks fine the way it is.

Reply:
After trying around a bit and looking for some inspiration I like this:
![grafik](https://github.com/valence-rs/valence/assets/75166328/46ea760a-507b-4f0e-b3ed-05c644d8778e)

Reply:
That looks good.